### ISSUE TITLE
 Standard names: Volume_fraction_of_water_in_soil_at_saturation (m3/m3)

### ISSUE THREAD

**Issue by I-Dhar** [cf-convention/discuss#251]
**Proposer's name** Imtiaz Dharssi

**Date** 10 July 2023

**- Term** volume_fraction_of_water_in_soil_at_saturation

**- Description** The proposed term corresponds to an essential soil physical parameter in land surface modelling and does not appear to be in the CF Standard Name Table. The standard table does contain a similar term called soil_porosity. However, the two terms are not the same and soil porosity can be slightly larger than the volume_fraction_of_water_in_soil_at_saturation. The proposed term is widely used for land surface modelling, as shown by the two links below
- ECMWF: Volumetric saturation of soil moisture https://codes.ecmwf.int/grib/param-db/?id=260216
- UKMO: VOL SMC AT SATURATION AFTER TIMESTEP https://reference.metoffice.gov.uk/um/stash/_m01s00i043

Related terms (with different meanings) already in the standard table are:
- soil_hydraulic_conductivity_at_saturation
- soil_suction_at_saturation
- volume_fraction_of_water_in_soil_at_wilting_point
- volume_fraction_of_water_in_soil_at_critical_point

I have followed the last two standard names above to derive the proposed term rather than using the ECMWF or UKMO wording.

**- Units** m3/m3

**Comment by JonathanGregory** [comment]
Thanks for your proposal, @I-Dhar and welcome to CF `discuss`. I think the proposal is good. I agree that you've followed existing patterns (except that the initial `v` should be lower-case). Jonathan

**Comment by I-Dhar** [comment]
Thanks Jonathan, I'm happy to accept your suggestion and have changed the initial letter to lower-case.

**Comment by feggleton** [comment]
Hi, thanks for your proposal. Just looking at volume_fraction_of_water_in_soil_at_wilting_point which is unitless, should this term be unitless too? Thanks

**Comment by feggleton** [comment]
We will also need a description, probably similar to  volume_fraction_of_water_in_soil_at_wilting_point, which is:

"Volume fraction" is used in the construction "volume_fraction_of_X_in_Y", where X is a material constituent of Y. It is evaluated as the volume of X divided by the volume of Y (including X). It may be expressed as a fraction, a percentage, or any other dimensionless representation of a fraction. The phrase "condensed_water" means liquid and ice. The wilting point of soil is the water content below which plants cannot extract sufficient water to balance their loss through transpiration.

**Comment by idharssi2020** [comment]
> Hi, thanks for your proposal. Just looking at volume_fraction_of_water_in_soil_at_wilting_point which is unitless, should this term be unitless too? Thanks

This quantity is a ratio of volumes, as is the soil wilting point, critical point and field capacity. Usually, in publications the units are given as (m3/m3) but could be unitless if that is preferred. I'm happy to accept advice on this matter.

**Comment by idharssi2020** [comment]
> We will also need a description, probably similar to volume_fraction_of_water_in_soil_at_wilting_point, which is:
> 
> "Volume fraction" is used in the construction "volume_fraction_of_X_in_Y", where X is a material constituent of Y. It is evaluated as the volume of X divided by the volume of Y (including X). It may be expressed as a fraction, a percentage, or any other dimensionless representation of a fraction. The phrase "condensed_water" means liquid and ice. The wilting point of soil is the water content below which plants cannot extract sufficient water to balance their loss through transpiration.

This is my suggested description
_"Volume fraction" is used in the construction volume_fraction_of_X_in_Y, where X is a material constituent of Y. The volume_fraction_of_water_in_soil_at_saturation is the volume fraction at which a soil has reached it's maximum water holding capacity._

Happy for others to expand the description if desired.

**Comment by I-Dhar** [comment]
I-Dhar and idharssi2020 are the same person. I accidentally replied with my other account.

**Comment by efisher008** [comment]
Hello @I-Dhar,

The units in the standard name have been updated to '1' (unitless). I hope that's fine. I have also updated the description to your suggested phrasing, with the addition of "_It is evaluated as the volume of X divided by the volume of Y (including X). It may be expressed as a fraction, a percentage, or any other dimensionless representation of a fraction._" (from @feggleton's suggestion).

The name and its full description can be viewed on the editor here: [https://cfeditor.ceda.ac.uk/proposal/5023/edit](url)
I think it is now in a position to be accepted, so if there are no further comments within 7 days, that will happen. Thanks again.

Best regards,
Ellie
